packages/react/popper/src/Popper.tsx

```diff
@@ -12,23 +12,20 @@ import type { Side, Align, Size, MeasurableElement, Merge } from '@radix-ui/util
  * Root level context
  * -----------------------------------------------------------------------------------------------*/
 
-type PopperContextValue = {
```

Comment:
Did you mean to leave the comment above?
Also, not sure I've paid attention to that in all the other ones 😬

Reply:
No i meant to remove that, think i did in the others (just missed them in the ones I opened today 😬 )